package.json

```diff
@@ -33,9 +33,9 @@
     "stacktrace-js": "~2.0.0",
     "superagent": "~3.8.2",
     "tap-finished": "~0.0.1",
-    "tap-parser": "~5.4.0",
-    "watchify": "~3.10.0",
-    "wd": "~1.5.0",
+    "tap-parser": "~7.0.0",
```

Comment:
Also gotta wait with this one. See https://github.com/airtap/airtap/pull/54 and [CHANGELOG.md](https://github.com/airtap/airtap/blob/3fc9053806d4d817360b7dda09b49aad609c801a/CHANGELOG.md#removed-3):

> We didn't update browserify to the latest version because it doesn't support IE9 or IE10. Nor did we update tap-parser to the latest since it doesn't work with PhantomJS.